Gen3_Source_injection.ipynb

```diff
@@ -0,0 +1,602 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #4.    from lsst.pipe.tasks.insertFakes import _add_fake_sources
</pre><p>here you import a private definition from another package. If possible, this should be avoided as private definitions might change without deprecation warnings etc. One way to do this is perhaps to ask the list.pipe.tasks.insertFakes developer to also add a public facing add_fake_sources() definition to signal them that this is used and should remain intact.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/LSST-strong-lensing/DC2-notebooks/pull/3/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='13'/>